services-js/commissions-search/src/FacetList.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from "react";
+import "./FacetList.css";
+
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+
+class FacetList extends Component {
+  render() {
+    const GET_AREAS = gql`
+      {
+        policyTypes {
+          id
+          name
+        }
+      }
+    `;
+
+    function Area(props) {
+      return (
+        <label className="cb">
+          <input
+            id="checkbox-area"
+            name={props.id}
+            type="checkbox"
+            value={props.areaValue}
+            className="cb-f"
+            checked={props.areaValue}
+            onChange={props.handleCheckChange}
+          />
+          <span className="cb-l">{props.name}</span>
+        </label>
+      );
+    }
+
+    const PolicyAreaComponents = () => (
+      <Query query={GET_AREAS}>
+        {({ loading, error, data }) => {
+          if (loading) return "Loading...";
+          if (error) return `Error! ${error.message}`;
+
+          return data.policyTypes.map(policy_area => (
+            <Area
+              key={policy_area.id}
+              id={policy_area.id}
+              name={policy_area.name}
+              areaValue={this.props.facets[policy_area.id]}
+              handleCheckChange={this.props.handleCheckChange}
+            />
+          ));
+        }}
+      </Query>
+    );
+
+    console.log(this.props.facets);
+    console.log(this.props.seats);
+    return (
+      <div className="co">
+        <input
+          id="collapsible"
+          type="checkbox"
+          className="co-f d-n"
+          aria-hidden="true"
+        />
+        <label htmlFor="collapsible" className="co-t">
+          Filter
+        </label>
+        <div className="co-b co-b--pl">
+          <div className="t--intro m-b200">Policy Area</div>
+          <div className="m-b300">
+            <PolicyAreaComponents />
+
+            <div className="t--intro m-b200">Open Seats</div>
+            <div className="m-b300">
+              <label className="cb">
+                <input
+                  id="checkbox-seats"
+                  name="seats-all"
+                  type="radio"
+                  value="null"
+                  checked={this.props.seats === "null"}
```

Comment:
I'd recommend either turning this into a boolean, or use more descriptive symbols (like "all" and "only-open" or something). Having `"null"` and `"true"` string literals is a recipe for confusion down the road, since `"null" !== null`